### DISCUSSION TITLE
chord optimization with scale_x or scale_z attributes

### DISCUSSION THREAD

**Discussion by shaheidula** [mdolab/dafoam#329] [General]
Hello Prof. Ping He, 
I am trying to use the scale attributes to optimize the chord, but it didn't work.  It used to work when I was using the previous versions.

**Comment by friedenhe** [comment]
Check this: https://github.com/DAFoam/tutorials/blob/a54a997ba7e45cf931f005952bcf1eeb557288bc/ADODG3_Wing/runScript_planform.py#L174
